opensrp-reporting/src/main/java/org/smartregister/reporting/impl/models/IndicatorModel.java

```diff
@@ -0,0 +1,40 @@
+package org.smartregister.reporting.impl.models;
+
+
+import org.smartregister.reporting.impl.views.IndicatorView;
+
+public class IndicatorModel {
```

Comment:
Maybe this is a *DisplayModel* ? Since it's essentially used when generating views?

Reply:
Not sure what you mean here. Do you mean renaming the class?.  You are definitely right, its data used by the view which displays the indicator. You can suggest a better name.

Reply:
Yeah, I meant renaming the class. My suggestion is `DisplayModel`